ui/src/main/main.tsx

```diff
@@ -46,6 +48,11 @@ const Main: React.FC<MainProps> = (props: MainProps) => {
             <div className={styles.main__content} qa-id="main-content-container">
                 <Routes>
                     <Route path="/" element={<FrontpageContainer />} />
+                    <Route path={'/publication-log'} element={<PublicationLog />} />
+                    <Route
+                        path={'/publication-log/:publicationId'}
+                        element={<PublicationDetailsContainer />}
```

Comment:
Uudet routet+näytettyjen komponenttien määritys riippuen urlista. AIemmin tällaisia urleja ei ollut, vaikuttavatko urlit järkeviltä?

Reply:
Omaan silmään nämä voisi ehkä olla vain `/publications/<id>` ja vaikka `/publications/log`, mutta ihan OK:t nämä näinkin. Kuvastaisi enemmän käsiteltävää dataa.

Reply:
No complaints. Näyttää minusta hyvältä näin, myös tuo Juusen ehdottama on ok.

Reply:
Muutettu hieman nätimmäksi Juusen ideaa mukaillen, nyt siis `/publications` johtaa julkaisulokin hakusivulle ja `/publications/<id>` tiettyyn julkaisuun.